### ISSUE TITLE
4.6 Alpha RC3 crashes to desktop when starting a song

### ISSUE THREAD

**Issue by bibusdax** [TeamRizu/OutFox#17]
I started the game, went to play selected standard, then in Ben Spears Galaxy I selected the song "I wish" and as soon as the arrows arrive on the screen it crashes. This is a new issue since RC2 with which I had no issues.
[crashinfo.txt](https://github.com/TeamRizu/OutFox/files/4870729/crashinfo.txt)

The computer used runs on Windows 10, is an ASUS Barebone with Intel Atom Chipset and some Nvidia Ion graphics.

**Comment by bibusdax** [comment]
[log.txt](https://github.com/TeamRizu/OutFox/files/4870879/log.txt)
[crashinfo.txt](https://github.com/TeamRizu/OutFox/files/4870880/crashinfo.txt)

**Comment by bibusdax** [comment]
It seems that the problem is associated with songs who have a video attached.

**Comment by animusyosho** [comment]
https://github.com/TeamRizu/OutFox/issues/18
Similar to the issue I just had, also when starting a song, also ben spiers galaxy, and also with a video.

**Comment by bibusdax** [comment]
Team Rizu, helped me on Discord and provided me a new swscale-5.dll which then did help.

TLDR:
The problem seems to be with FFMPEG where any developer can select among two versions:
LGPL reduced version which allows use in commercial products and does not contain delicated patented stuff.
GPL full version which is risky to be used in commercial products as it may contain Frauenhofer patented stuff. (However many of these patents do have expired by now)
And more specific, the issue is only on windows where they use the Binary from Zeranoe who puts it under the more strict GPL 3.0 and LGPL 3.0 version. However I would legally doubt that he is allowed to deviate from the source code license.

**Comment by bibusdax** [comment]
Btw. the same crash happens on Kubuntu with the Ben Speirs' SPEIRMIX GALAXY songs.

**Comment by Scraticus** [comment]
This will be fixed in our next Release, it's an inherited FFMPEG bug, so apologies this has affected your experience.

**Comment by Scraticus** [comment]
This is fixed as of 4.6.0 Final grab the new release!
